src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -469,6 +470,32 @@ public function register(Application $app)
 
                 $listener->addHandler(new SessionLogoutHandler());
 
+                $removeCookies = isset($options['remove_cookies']) ? $options['remove_cookies'] : null;
+                if (null !== $removeCookies) {
```

Comment:
The `$removeCookies` is not necessary at all.
Should be directly `if (isset($options['remove_cookies'])) {` and then `foreach ($options['remove_cookies'])` ...